src/prefect/tasks/files/operations.py

```diff
@@ -171,3 +171,50 @@ def run(self, path: Union[str, Path] = "") -> None:
             path.unlink()
         else:
             path.rmdir()
+
+
+class Glob(Task):
+    """
+    This task returns the content of a given directory matching a given pattern within the file system.
+    Args:
+        - path (Union[str, Path], optional): directory path
+        - pattern (str, optional): glob pattern, defaults to *
+        - **kwargs (dict, optional): additional keyword arguments to pass to the
+            Task constructor
+    """
+
+    def __init__(
+        self,
+        path: Union[str, Path] = "",
+        pattern: str = "*",
+        **kwargs: Any,
+    ):
+        self.path = path
+        self.pattern = pattern
+        super().__init__(**kwargs)
+
+    @defaults_from_attrs("path", "pattern")
+    def run(
+        self,
+        path: Union[str, Path] = "",
+        pattern: str = "*",
+    ) -> List[Path]:
+        """
+        Task run method.
+
+        Args:
+            - path (Union[str, Path], optional): directory path
+            - pattern (str, optional): glob pattern, defaults to *.
```

Comment:
```suggestion
            - pattern (str, optional): glob pattern, defaults to "*" which lists all files in the path
```
or does it list all objects including directories?